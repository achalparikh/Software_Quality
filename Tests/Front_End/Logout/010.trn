10_admin_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
10_admin_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _