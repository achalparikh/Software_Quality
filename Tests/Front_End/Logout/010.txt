login
admin
logout
deposit
00001
0.01
login
logout