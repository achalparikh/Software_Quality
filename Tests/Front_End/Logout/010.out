Cannot perform any transactions, there is currently no user logged in.